tests/test_mock_controller.py

```diff
@@ -97,18 +99,30 @@ async def tearDown(self):
             self.writer.close()
 
     async def test_command_does_not_exist(self):
+        # Temporarily disable validation exceptions for the unit test.
+        # Validation of the commands should be done by the client and the
+        # simulator has such validation built in.
+        MTDome.encoding_tools.validation_raises_exception = False
```

Comment:
Do you really need this? It might be better to keep it True

Reply:
In the three particular test cases where I added that line, it indeed is needed since the simulator needs to do the validation and not the encoding tools, as I hoped the comment would explain.